example/App.js

```diff
@@ -12,15 +12,30 @@ import {
 import DateTimePicker from '@react-native-community/datetimepicker';
 import {Header, Colors} from 'react-native/Libraries/NewAppScreen';
 import React, {useState} from 'react';
+import {Picker} from 'react-native-windows';
 import moment from 'moment';
+import {DAY_OF_WEEK} from '../src/constants';
 
 const App = () => {
-  const [date, setDate] = useState(new Date(1598051730000));
```

Comment:
is removing the `1598051730000` part a problem for some reason? 

The end-to-end tests for ios and android, albeit currently not executed, rely on this. I'm planning to reintroduce them soon, so keeping this would make that a little easier. Thanks!

Reply:
I removed it so that the 'text placeholder' prop in the windows version gets displayed initially, before the control is clicked and a date is selected.
![image](https://user-images.githubusercontent.com/12816515/81098488-49203780-8ebe-11ea-9e57-8b21d93bf094.png)

I can restore it and add a 'windowsDate' prop to achieve the same thing, though.